core/src/zeit/content/article/article.py

```diff
@@ -574,6 +574,18 @@ def get_dependencies(self):
         return ()
 
 
+def _categorize_by_ingredients_diet(ingredients):
+    diets = {
+        zeit.wochenmarkt.sources.ingredientsSource(None).factory.find(None, i).diet
+        for i in ingredients
+    }
```

Comment:
```suggestion
    source = zeit.wochenmarkt.sources.ingredientsSource(None)
    diets = {source.find(x).diet for x in ingredients}
```

Die Zope Source API ist... speziell.